### ISSUE TITLE
Twitch api v5 support. Remove deprecated api methods.

### ISSUE THREAD

**Issue by kobel169** [dustinlakin/twitch-rb#30]

**Comment by ponny** [comment]
We've done this task: https://github.com/Crowd9/twitch-rb

Would take extra work to fix the specs, docs, and migration path for old gem versions to do a pull request.  Given that our pull from September is still outstanding, I don't think we'll bother though.

**Comment by kobel169** [comment]
Oh, cool. TY, I migrate to your version and waits for her in the mainstream.

**Comment by dustinlakin** [comment]
@ponny I added you as a collaborator, I am really sorry. I haven't had much time to help out and maintain this project :/

**Comment by ponny** [comment]
That's fair enough.  Sorry if my comment came across as snarky.  I'll see about getting a pull done but no guarantees :-)

**Comment by dustinlakin** [comment]
Nope, not at all, I just haven't been able give this project any attention at all.

**Comment by AlexWayfer** [comment]
Maybe I'll also try to actualize this gem and create a PR, but only one and huge (there are too many changes needed for a first sight).

**Comment by AlexWayfer** [comment]
Done in #34. @dustinlakin, can you take a look please? I see that @ponny is not collaborator right now.
